src/views/fullcalendar/FullCalendarDemo.vue

```diff
@@ -29,8 +29,9 @@ export default {
         return {
             options: {
                 plugins:[dayGridPlugin, timeGridPlugin, interactionPlugin],
-                defaultDate: '2019-01-01',
-                header: {
+                initialView: 'dayGridMonth', 
+                initialDate: '2019-01-01',
```

Comment:
Can we change this to the current date? I think it's suitable more in most cases.